tests/features/user/register.feature

```diff
@@ -35,7 +29,9 @@ Feature: User registration
       | body      | The Joinup Support Team created your account on Joinup. Please log in through the following link in order to set your password. |
     And the account for SuperUser should be active
 
-  @email
+  # This test should be converted for EU Login but seems incomplete and is
+  # probably no longer applicable and can be removed in ISAICP-5760.
+  @email @wip
```

Comment:
Why this TODO as we have ISAICP-5760?

Reply:
Because of the `@wip`, when the test is disabled it will not fail.